### ISSUE TITLE
Change notifications icon

### ISSUE THREAD

**Issue by nicofrand** [cozy/cozy-mobile#302]
If the notification is not a reminder from cozy-calendar, I'd like the notification icon to be cozy's icon (a cloud) instead of the bell.

**Comment by Darcidride** [comment]
It's a good idea

But... something like this ![](http://i.imgur.com/iBv2vjt.png) (arrow down)
or like this ![](http://i.imgur.com/2nbcuZy.png) (arrow circle loop)

**Comment by nicofrand** [comment]
I disagree. The notifications are hère to tell me something happened, not to tell me that cozy-mobile is synchronizing.

**Comment by Darcidride** [comment]
So, your point of view is only a cloud instead of the actual bell ?

**Comment by nicofrand** [comment]
Either a cloud or the app's icon (app being the one installed on cozy)

Le mer. 14 déc. 2016 à 19:58, Darcidride <notifications@github.com> a
écrit :
